File: FinalProject/MovieApi/movies/src/main/java/project/movies/models/Movie.java
package project.movies.models;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Table
@Getter
@Setter
public class Movie implements Serializable {

    @Id
    private Long id;
    private String title;
    private String category;
    private Integer year;
    private String cast;
    private String director;
    private String story;
    private Double price;
}

File: FinalProject/MoviesFront/src/app/Movie.ts
export class Movie {

    id: number;
    title: string;
    category: string;
    year: number;
    cast: string;
    director: string;
    story: string;
    price: number;

    constructor(id: number, title: string, category: string, year: number, cast: string, director: string, story: string, price: number) {
        this.id = id;
        this.title = title;
        this.category = category;
        this.year = year;
        this.cast = cast;
        this.director = director;
        this.story = story;
        this.price = price;
    }
}